From 8d7d9fb6e89644fdf2a97e866ec0051f94434f77 Mon Sep 17 00:00:00 2001
From: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
Date: Fri, 13 Feb 2015 19:01:16 +0530
Subject: [PATCH] arch/x86: Add HARDIRQS_SW_RESEND config for cherrytrail

Adding HARDIRQS_SW_RESEND config to avoid some wakeup irqs
lost when wakeup from S3, such as pmic/charger irq miss and
power button can't wakeup system by short pressing.

Change-Id: I2b6cc38ba91e67f11b2a1c7a75ca382a802cd0ee
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1017
Signed-off-by: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
---
 arch/x86/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index b656f03..9978183 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -86,6 +86,7 @@ config X86
 	select HAVE_USER_RETURN_NOTIFIER
 	select ARCH_BINFMT_ELF_RANDOMIZE_PIE
 	select HAVE_ARCH_JUMP_LABEL
+	select HARDIRQS_SW_RESEND
 	select ARCH_HAS_ATOMIC64_DEC_IF_POSITIVE
 	select SPARSE_IRQ
 	select GENERIC_FIND_FIRST_BIT
-- 
1.9.1

